packages/surveycto/src/Utils.js

```diff
@@ -0,0 +1,59 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+const addBasicAuth = (configuration = {}, headers) => {
+  const { username, password } = configuration;
+  if (username && password) {
+    Object.assign(headers, makeBasicAuthHeader(username, password));
+  }
+};
+
+const buildUrl = (configuration = {}, path) => {
+  const { servername, apiVersion = 'v1' } = configuration;
+  if (!servername) throw 'Please specify servername in your credentials';
+  return `https://${servername}.surveycto.com/api/${apiVersion}${path}`;
+};
+
+export const formDate = date => Math.floor(new Date(date).getTime() / 1000);
```

Comment:
Can we give this a better name, like `makeSurveyCTODate`? Or something more descripive?